### ISSUE TITLE
Apply su-underline link style to About and Home pages

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/rialto-web#144]
There are a number of links on the About and Home pages missing the `class="su-underline"` link style (or the style is not displaying correctly): https://sul-dlss.github.io/component-library/links/.

<img width="2922" height="3372" alt="Image" src="https://github.com/user-attachments/assets/e40328a9-b999-41b7-aa93-5fd0d6bf1656" />

<img width="2922" height="2528" alt="Image" src="https://github.com/user-attachments/assets/8def6633-6514-4d38-a81e-b483a98e3dd3" />
